src/services/UserService.ts

```diff
@@ -15,6 +15,7 @@ const unlink = promisify(_unlink);
 
 
 export type UserDataToCreate = Pick<User, 'forename' | 'surname' | 'email' | 'password'>;
+export interface PasswordResetDataToCreate { password: string }
```

Comment:
```suggestion
export interface PasswordResetData { password: string }
```

I think the `ToCreate` for this specific type can be confusing, it exists for the other ones as they define the required data needed to create an entity (e.g. to create a user, to create a profile). Since the password isn't an entity in itself, I think it is safe to omit this.